notes-flow.txt

```diff
@@ -0,0 +1,23 @@
+common:
+  corrId - random BS, used as CbNonce
+  entityId - p2r tlsUniq
```

Comment:
don't think we have to use tlsuniq as session ID, can be just random? Or will it be included as sessionId in e2e encrypted commands to prevent replays?